packages/core/lib/commands/console/meta.js

```diff
@@ -27,6 +43,8 @@ module.exports = {
           "in your project's truffle-config.js."
       }
     ],
-    allowedGlobalOptions: ["network", "config"]
+    // The 'network' option is excluded from here and added manually above as a workaround to combine its usage
+    // instructions with url to show that they are mutually exclusive.
+    allowedGlobalOptions: ["config"]
```

Comment:
We don't communicate `config`'s behavior with `--url`

Reply:
You are right @cds-amal .`config` option's behavior is not related to `--url` option. `config` is just added to the global options if we want to use a config file which is outside of a truffle project.